### PR TITLE
Adding missing folder nestedtemplates to artifact

### PR DESCRIPTION
**Pull request by fepere**
### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use a parameter for resource locations with the defaultValue set to resourceGroup().location
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values)
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog

*
*
*

### CODE REVIEW

**Comment by fepere** [comment]
added 2 more changes

File: 301-storage-spaces-direct-md-zones/azuredeploy.json [hunk 1]
```diff
@@ -111,7 +111,7 @@
       "metadata": {
         "description": "Location of resources that the script is dependent on such as linked templates and DSC modules"
       },
-      "defaultValue": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/301-storage-spaces-direct-md-zones"
+      "defaultValue": "https://raw.githubusercontent.com/fepere/azure-quickstart-templates/master/301-storage-spaces-direct-md-zones"
```
Comment (bmoore-msft) [inline 301-storage-spaces-direct-md-zones/azuredeploy.json]: revert this change

Comment (fepere) [reply 301-storage-spaces-direct-md-zones/azuredeploy.json]: Hello,
I revert back, I was made that due the previous was not working, so I can moved forward. Thx
